tools/deb/istio-iptables.sh

```diff
@@ -299,9 +299,11 @@ iptables -t nat -N ISTIO_OUTPUT
 # Jump to the ISTIO_OUTPUT chain from OUTPUT chain for all tcp traffic.
 iptables -t nat -A OUTPUT -p tcp -j ISTIO_OUTPUT
 
-# Redirect app calls to back itself via Envoy when using the service VIP or endpoint
-# address, e.g. appN => Envoy (client) => Envoy (server) => appN.
-iptables -t nat -A ISTIO_OUTPUT -o lo ! -d 127.0.0.1/32 -j ISTIO_REDIRECT
+if [ -z "${DISABLE_REDIRECTION_ON_LOCAL_LOOPBACK-}" ]; then
```

Comment:
Should the variable inside the ${} be "DISABLE_REDIRECTION_ON_LOCAL_LOOPBACK-" or "DISABLE_REDIRECTION_ON_LOCAL_LOOPBACK"?

Reply:
This is still 'on by default' - please swap it to off by default.

Reply:
And maybe call it 'REDIRECT_LOOPBACK'... 

It is too risky to make on-by-default iptables changes in the last minute, given the lack of tests ( and difficulty to manually test ).

Reply:
By default, it's off. DISABLE_REDIRECTION_ON_LOCAL_LOOPBACK does not exist by default so 
${DISABLE_REDIRECTION_ON_LOCAL_LOOPBACK-} evaluates to an empty string. Without appending the dash in the end, the script will fail since the variable is unset.